language/tools/move-cli/src/main.rs

```diff
@@ -775,7 +778,33 @@ fn main() -> Result<()> {
                 *dry_run,
             )
         }
-        Command::Test { path, track_cov } => test::run_all(
+        Command::Test {
+            path,
+            track_cov: _,
+            create: true,
+        } => {
+            let dirs = ["src", "src/modules", "src/scripts"];
```

Comment:
I think we have a `pub const DEFAULT_SOURCE_DIR: &str = "src";` in `lib.rs`. Not sure if it will be better to use the const instead of hardcoding it here.

Reply:
Good spot. I've updated. I've also moved this logic in to the `test` module since that seems to be a better place for it. I also added a check so that we don't overwrite an existing directory/file when create a scaffold.